.husky/pre-commit

```diff
@@ -9,12 +9,12 @@ echo 'Running pre-commit hook...'
 chronic typos .
 
 # lint and format for core contracts and typescript codes
-(cd core && chronic pnpm lint)
+(cd core && chronic pnpm lint && pnpm format)
 
 # lint and format for relayer codes
 (cd relayer && chronic mage lint && chronic go fmt ./...)
 
 # cargo fmt
-#(cd parachain && SKIP_WASM_BUILD= cargo fmt --all)
+(cd parachain && SKIP_WASM_BUILD= cargo +nightly fmt --all)
```

Comment:
This gives me an error:
```
error: no such command: `+nightly`

	Cargo does not handle `+toolchain` directives.
	Did you mean to invoke `cargo` through `rustup` instead?
husky - pre-commit hook exited with code 101 (error)
```

Looks like it works outside of Nix, so likely something the overlay does. Have you tested this in a Nix shell?

Reply:
Happy to merge if we remove `nightly` for now. This skips a lot of checks, but adding `nightly` breaks the pre-commit hook in a Nix shell.
```suggestion
(cd parachain && SKIP_WASM_BUILD= cargo fmt --all)
```